src/pages/task/executionDropdown/ExecutionSelector.tsx

```diff
@@ -63,7 +63,7 @@ export const ExecutionSelect: React.VFC<ExecutionSelectProps> = ({
           >
             <ExecutionInfo>
               <StyledTaskStatusIcon status={singleExecution.status} />
-              <StyledBody title={optionText}>{optionText}</StyledBody>
+              <StyledBody>{optionText}</StyledBody>
```

Comment:
I'm not sure if we should remove the `title` attribute — the reason it was added is because the text often gets cut off in the execution dropdown

<img width="320" alt="Screen Shot 2022-10-26 at 8 16 54 PM" src="https://user-images.githubusercontent.com/47064971/198162171-0e26489f-166e-45a3-9ecf-dece80c7ee3d.png">